addon/components/paper-dialog-inner.js

```diff
@@ -18,32 +18,41 @@ export default Component.extend(Translate3dMixin, {
 
   onTranslateFromEnd() {
     if (this.get('focusOnOpen')) {
-      let toFocus = this.$('[autofocus]').last();
-      if (toFocus.length === 0) {
-        toFocus = this.$('md-dialog-actions button').last();
+      let focusableElements = this.element.querySelectorAll('[autofocus]');
+      let toFocus = focusableElements[focusableElements.length - 1];
+      if (!toFocus) {
+        let focusableButtons = this.element.querySelectorAll('md-dialog-actions button');
+        toFocus = focusableButtons[focusableButtons.length - 1];
+      }
+      if(toFocus) {
+        toFocus.focus();
       }
-      toFocus.focus();
     }
   },
 
-  onTranslateToEnd($origin) {
-    if ($origin) {
-      $origin.focus();
+  onTranslateToEnd(origin) {
+    if (origin) {
+      origin.focus();
     }
   },
 
   didInsertElement() {
     this._super(...arguments);
     this.checkContentOverflow();
     // content overflow might change depending on load of images inside dialog.
-    let images = this.$().find('img');
-    images.on(`load.${this.elementId}`, run.bind(this, this.checkContentOverflow));
+    let imageElements = this.element.querySelectorAll('img');
+    imageElements.forEach( image => {
+      image.addEventListener('load', run.bind(this, this.checkContentOverflow))
+    });
+  
   },
 
   willDestroyElement() {
     this._super(...arguments);
-    let images = this.$().find('img');
-    images.off(`load.${this.elementId}`);
+    let imageElements = this.element.querySelectorAll('img');
+    imageElements.forEach( image => {
+      image.removeEventListener('load', run.bind(this, this.checkContentOverflow))
```

Comment:
I think that this `removeEventListener` is not working as expected, because `run.bind` returns a new function, different from the one used on the previous `window.addEventListener`.

I think we need to do the same trick that you did on #995, which is basically:

```js
this._checkContentOverflow = run.bind(this, this. checkContentOverflow);
imageElements.forEach((image) => {
  image.addEventListener('load', this._checkContentOverflow);
});
```

and later

```js
imageElements.forEach((image) => {
  image.removeEventListener('load', this, this._checkContentOverflow);
});
```